src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,75 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import imageUrl from "../../static/images/card/card.png";
+import * as styles from "./Card.module.css";
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+interface CardContentProps {
+  header: string;
```

Comment:
**nitpick:** We should not need these props. They are restrict us from inserting anything else the these two things. `CardContent` should accept only `PropsWithChildren<{}>` as interface